lifecycle-operator/chart/templates/keptnevaluation-crd.yaml

```diff
@@ -79,6 +79,8 @@ spec:
                 type: string
               evaluationDefinition:
                 type: string
+              failAction:
```

Comment:
Please remove the failAction parameter from the v1beta1 version

Reply:
okay